stdlib/public/core/ErrorType.swift

```diff
@@ -35,7 +35,7 @@ import SwiftShims
 ///
 ///     enum IntParsingError: Error {
 ///         case overflow
-///         case invalidInput(String)
+///         case invalidInput(Character)
```

Comment:
The initializer using this in the next section is also using `s` as though the variable it's validating is a `String` . Should that either be `c` or something more descriptive @natecook1000?

Reply:
Thanks for catching this, @kandelvijaya! I agree with @CodaFi that we should update the code sample that throws an error—do you want to take a crack at that? It could just be changing `s` to `c`, or if you want to modify the sample so it gives more of a hint of processing character by character that could be even better.

Reply:
Thanks @natecook1000 for the pointer. I will update the `s` to `c`.